From 4c7229f1754e62587b812f90a4e004763159af6d Mon Sep 17 00:00:00 2001
From: olivier moysan <olivier.moysan@st.com>
Date: Wed, 26 Jul 2017 11:50:29 +0200
Subject: [PATCH 202/529] ASoC: Update wm8994 codec config

Allow selection of Wolson wm8994 codec from defconfig.

Signed-off-by: olivier moysan <olivier.moysan@st.com>
Change-Id: Iaf5cddb0d0af22730a8c564c19e8645ed17ef545
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/279000
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Olivier MOYSAN <olivier.moysan@foss.st.com>
Reviewed-by: Olivier MOYSAN <olivier.moysan@foss.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 sound/soc/codecs/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/sound/soc/codecs/Kconfig
+++ b/sound/soc/codecs/Kconfig
@@ -1981,7 +1981,7 @@ config SND_SOC_WM8993
 	depends on I2C
 
 config SND_SOC_WM8994
-	tristate
+	tristate "Wolfson Microelectronics WM8994 codec"
 
 config SND_SOC_WM8995
 	tristate
